Formula/fpc.rb

```diff
@@ -14,17 +13,20 @@ class Fpc < Formula
   end
 
   resource "bootstrap" do
-    url "https://downloads.sourceforge.net/project/freepascal/Bootstrap/3.0.0/x86_64-macosx-10.7-ppcx64.tar.bz2"
-    sha256 "a67ef5def356d122a4692e21b209c328f6d46deef4539f4d4506c3dc1eecb4b0"
+    url "https://downloads.sourceforge.net/project/freepascal/Mac%20OS%20X/3.0.4/fpc-3.0.4a.intel-macosx.dmg"
```

Comment:
I had to swap bootstrap compiler with full distribution because FPC officially supports bootstrapping only from the previous release. I tried using 3.0.0 but that failed and at the same time there is no bootstrapping distribution of 3.0.4 for macOS 😓